src/CallingExtensions.js

```diff
@@ -129,6 +131,10 @@ class CallingExtensions {
     this.iFrameManager.sendMessage(message);
   }
 
+  logDebugMessage(message) {
+    this.iFrameManager.logDebugMessage(prefix, "Console Debug", message);
+  }
```

Comment:
Used `Console Debug` to differentiate this message from HS and Third Party events as I couldn't think of another identifier. Feel free to throw suggestions

Reply:
Console Debug can be confusing. May be 'CallExt Debug'.

Reply:
Oops. I dint see the prefix defined already. Isnt prefix sufficient enough to differentiate?

Reply:
The prefix is used in other messages as well
```
this.logDebugMessage(prefix, "To HubSpot", type, message);
this.logDebugMessage(prefix, "From HubSpot", type, { data });
```

Reply:
Hmmm. May be, we can add provider name here. Like aircall, justcall. would that be of help?

Reply:
We're using `logDebugMessage` in demo minimal js to log an incoming contact and leveraging the prefix to filter out Calling Extensions messages. However, since this isn't a message event to or from HubSpot, I'm thinking that something like `Debug` would be useful for knowing the purpose of the logging.

This message wouldn't show up in any other apps unless they were to use `logDebugMessage()` as well and in such cases, I'm not sure if the provider name would be helpful.